docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,442 @@
+
+First, we join user data with cluster labels, create separate DataFrames for each cluster, generates activity histograms for each cluster, and then concatenates these histograms into a single visualization.
+
+```python
+# activity histograms by cluster
+user_df = user_df.set_index("id").join(label_df)
+
+by_cluster_data = {
+    label: user_df[user_df["cluster_label"] == label]
+    for label in np.unique(hdbscan.labels_)
+}
+
+activity_histograms = [
+    alt.Chart(user_df_part)
+    .mark_bar()
+    .encode(x=alt.X("activity", bin=True), y="count()")
+    .properties(title=f"Activity histogram for cluster {label}")
+    for label, user_df_part in by_cluster_data.items()
+]
+
+alt.hconcat(*activity_histograms)
+```
+
+![histograms of clusters](../assets/use_cases/user_acquisition_analytics/cluster_histograms.png)
+
+From our histograms, we can observe that:
+
+- cluster -1 has an unusually large number of active users (typically, there are *very* few highly active application users)
+- cluster 2 and cluster 3 users are quite similar, but low activity
+- cluster 0 has the highest proportion of medium activity users
+- cluster 1 users are moderately active, with relatively few very high activity users and very few low activity users
+
+To see the distribution of ad_creatives across different clusters, we create a DataFrame that shows each ad_creative's count value within each cluster:
+
+```python
+pd.DataFrame(user_df.groupby("cluster_label")["ad_creative"].value_counts())
+```
+
+![ad creatives distribution across clusters](../assets/use_cases/user_acquisition_analytics/ad_creatives-per-cluster.png)
+
+observations:
+
+- outliers (cluster -1, highly active users) clicked on ad_creatives from both campaigns (as expected)
+- cluster 3 clicked on only one distinct ad_creative - from the influencer based campaign
+- clusters 0 and 2 clicked on only two distinct influencer based creatives
+- cluster 1 clicked on both campaigns' ad_creatives, but more on the first (non-influencer) campaign
+
+Now, let's get some descriptive stats for our signup dates to help us interpret our clusters' behavior further. 
+
+```python
+user_df["signup_datetime"] = [
+    datetime.fromtimestamp(ts) for ts in user_df["signup_date"]
+]
+desc = user_df.groupby("cluster_label")["signup_datetime"].describe()
+for col in desc.columns:
+    if col == "count":
+        continue
+    desc[col] = [dt.date() for dt in desc[col]]
+desc
+```
+
+![signup date per cluster group](../assets/use_cases/user_acquisition_analytics/signup_date-per-cluster.png)
+
+observations...
+
+- outliers' (cluster -1) have signup dates that are scattered
+- cluster 1's signups mostly (75%) came from clicks on the first campaign's ad_creatives
+- clusters 0, 2, and 3 signed up in response to the new (influencer-augmented) campaign only
+
+
+![summary of outcomes](../assets/use_cases/user_acquisition_analytics/summary.png)
+
+## In sum
```

Comment:
@morkapronczay

Reply:
I have not formulated takeaways myself from the clusters - probably along the lines of "users reacting to message x have a higher chance of becoming our active users so that is creative we should double down on" ... etc

do you think you could formulate some of these from the tables?